tests/unit/swagger/templateHelpers.spec.ts

```diff
@@ -4,6 +4,62 @@ import * as templateHelpers from './../../../src/routeGeneration/templateHelpers
 
 describe('templateHelpers', () => {
 
+    afterEach(() => {
+        // clean up global models
+        Object.keys(templateHelpers.models).forEach(k => {
```

Comment:
why not simple reset it to `{}`?

Reply:
This is not possible in typescript because exported globals are read-only from outside the module.

I agree this is a bit hacky.

I think a cleaner approach would be to avoid the global and wrap all the validation functions into a service class that takes the model as constructor argument. But I did not want to change so much in your code.